src/v0/util/index.js

```diff
@@ -2052,17 +2052,24 @@ const getAuthErrCategoryFromStCode = (status) => {
   }
   return '';
 };
+
 const isValidInteger = (value) => {
   if (Number.isNaN(value) || !isDefinedAndNotNull(value)) {
     return false;
   }
-  // const valueToString = `${value}`;
+  const valueToString = `${value}`;
   // Use a regular expression to check if the string is a valid integer or a valid floating-point number
-  return typeof value === 'string' ? /^-?\d+(\.\d+)?$/.test(`${value}`) : false;
+  return /^-?\d+$/.test(valueToString);
+};
+const validateEventType = (event) => {
```

Comment:
seems like repeated

Reply:
we are converting the input value to string and then checking it through regex expression to avoid multiple type checks